uint8 NONE = 0
uint8 INIT = 1
uint8 CONNECT = 2
uint8 DISCONNECT = 3
uint8 START = 4
uint8 PAUSE = 5
uint8 RESUME = 6
uint8 STOP = 7